01_fib/harirakr/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int) []int {
+	if n >= 93 || n <= -93 {
+		fmt.Fprintln(out, "fib: Range out of bounds; takes input > -93 and < 93")
+		return nil
+	}
+	series := make([]int, 0)
```

Comment:
`make([]int, 0, n )` you know the capacity.

Reply:
Done